tests/case/helpers.py

```diff
@@ -5,24 +5,25 @@
 
 class CaseHelper:
     _id = randint(1, 999999)
-    _deed_id = 1
     _conveyancer_id = 1
     _status = "the status"
     _last_updated = datetime.today()
     _created_on = datetime.today()
 
     @staticmethod
-    def _create_case_db():
+    def _create_case_db(deed_id=None):
```

Comment:
Personally I think there should be a separation of concerns here.

Either the _create_case_db should both create the deed and the case 
or 
there should be separate methods for creating a case, creating a deed, and updating the case with the deed_id

Its not very reusable the way it currently is. Also that _create_case_db is an awful name, I know I named it that but it could use a better name like _create_case_in_db or something similar. 

Also if the deed is being added then the name should reflect that too in my opinion.